cpp/mrc/include/mrc/edge/edge_holder.hpp

```diff
@@ -37,6 +37,7 @@
 #include <memory>
 #include <mutex>
 #include <optional>
+#include <sstream>  // for std::stringstream
```

Comment:
```suggestion
```

Reply:
How is this not needed? if I am using `std::stringstream`?

Reply:
Oh, I see now